remotestorage-module.js

```diff
@@ -1,41 +1,36 @@
-// This variable will be reused in the other files
-// Enable change events for changes in the same browser window
-var remoteStorage = new RemoteStorage(
-  {changeEvents: { local: true, window: true, remote: true, conflicts: true }}
-);
-
-remoteStorage.addModule({name: 'myfavoritedrinks', builder: function(privateClient) {
-  privateClient.declareType('drink', {
-    type: 'object',
-    properties: {
-      name: { type: 'string' }
-    },
-    required: ['name']
-  });
+window.myfavoritedrinks = {
```

Comment:
1. Our convention here is UppercaseCamelCase.
2. You don't need to specify `window`. Any `var` outside of another scope will be attached to the global object, which is `window` in browsers.